website/docs/d/repositories.html.markdown

```diff
@@ -25,10 +26,11 @@ data "github_repositories" "example" {
 The following arguments are supported:
 
 * `query` - (Required) Search query. See [documentation for the search syntax](https://help.github.com/articles/understanding-the-search-syntax/).
-
-* `sort` - (Optional) Sorts the repositories returned by the specified attribute. Valid values include `stars`, `fork`, and `updated`. Defaults to `updated`.
+* `sort` - (Optional) Sorts the repositories returned by the specified attribute. Valid values include `stars`, `fork`, `name`, and `updated`. Defaults to `updated`.
```

Comment:
`name` isn't actually present in the validation function above. Is it allowed by the API?

Reply:
I wrote this a long time ago so I'm trying to see why I modified that, but I believe it is a mistake that probably carried over as at some point I was going to add a resort to allow name but decided against it to keep it the same as current. I will update to resolve, removing 'name'.